python/pyarrow/gandiva.pyx

```diff
@@ -186,10 +186,6 @@ cdef class Projector(_Weakrefable):
         self.pool = pool
         return self
 
-    @property
-    def llvm_ir(self):
-        return self.projector.get().DumpIR().decode()
```

Comment:
I have to remove the `llvm_ir` API from python binding, since the `Configuration` is not exposed via the `make_projector` and `make_filter` APIs, but we may consider adding it if we can expose the `Configuration` as well

Reply:
Can we keep this? This was introduced by https://github.com/apache/arrow/pull/6417 .